src/components/ebay-notice/migrator.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @description
+ * Migrates each notice to different types
+ */
+
+// Types of buttons that if they are direct children of root will be wrapped in footer
+const buttonTypes = ['ebay-button', 'ebay-menu-button'];
+
+function setAttribute(el, oldName, newName) {
+    if (el.hasAttribute(oldName)) {
+        const attribute = el.getAttribute(oldName);
+        attribute.name = newName;
+    }
+}
+
+function migrator(el, context) {
+    const type = el.hasAttribute('type') && el.getAttributeValue('type').value;
+
+    if (type === 'inline') {
+        context.deprecate('<ebay-notice type="inline"/> has been renamed to <ebay-inline-notice>');
+        el.setTagName('ebay-inline-notice');
+    } else if (type === 'section') {
+        context.deprecate('<ebay-notice type="section"/> has been renamed to <ebay-section-notice>');
+        el.setTagName('ebay-section-notice');
+    } else if (type === 'window') {
+        context.deprecate('<ebay-notice type="window"/> has been renamed to <ebay-window-notice>');
+        el.setTagName('ebay-window-notice');
+    } else {
+        // Default is page notice
+        context.deprecate('<ebay-notice/> has been renamed to <ebay-page-notice>');
+        el.setTagName('ebay-page-notice');
+    }
+
+    setAttribute(el, 'a11y-heading-text', 'attyText');
```

Comment:
Prefer `el.renameAttribute(oldName, newName)`.

Reply:
It doesn't exist 👎 
Throws an error

Reply:
Lame. I thought this was exposed on custom elements but I was wrong.

Either way lets make this more clear and make our helper called `renameAttribute` or something like that, and also use `removeAttribute()` and `setAttribute()`.

Reply:
I will create a common method once dialog and this get merged.